cli/src/cmd/bind.rs

```diff
@@ -0,0 +1,184 @@
+use crate::cmd::Cmd;
+
+use clap::{Parser, ValueHint};
+use ethers::contract::MultiAbigen;
+use foundry_config::{
+    figment::{
+        self,
+        error::Kind::InvalidType,
+        value::{Dict, Map, Value},
+        Metadata, Profile, Provider,
+    },
+    impl_figment_convert, Config,
+};
+use serde::Serialize;
+use std::{fs, path::PathBuf};
+
+impl_figment_convert!(BindArgs);
+
+#[derive(Debug, Clone, Parser, Serialize)]
+pub struct BindArgs {
+    #[clap(
+        help = "The project's root path. By default, this is the root directory of the current Git repository or the current working directory if it is not part of a Git repository",
+        long,
+        value_hint = ValueHint::DirPath
+    )]
+    #[serde(skip)]
+    pub root: Option<PathBuf>,
+
+    #[clap(
+        help = "Path to where the contract artifacts are stored",
+        long = "out",
+        short,
+        value_hint = ValueHint::DirPath
+    )]
+    #[serde(rename = "out", skip_serializing_if = "Option::is_none")]
+    pub out_path: Option<PathBuf>,
+
+    #[clap(
+        help = "A directory at which to write the generated rust crate",
+        long = "bindings-root",
+        value_hint = ValueHint::DirPath
+    )]
+    #[serde(skip)]
+    pub bindings_root: Option<PathBuf>,
+
+    #[clap(
+        long = "crate-name",
+        help = "The name of the rust crate to generate. This should be a valid crates.io crate name. However, it is not currently validated by this command."
+    )]
+    #[serde(skip)]
```

Comment:
Should this be skipped? Maybe we should add them in the foundry.toml config, so that one can pre-specify the crate name etc. in the config?

Reply:
given that it's not already in the config, and I don't understand the config very well, I opted to skip it 😅